src/test/java/org/folio/rest/impl/LedgerFYTest.java

```diff
@@ -0,0 +1,77 @@
+package org.folio.rest.impl;
+
+import static org.folio.rest.utils.TestEntities.FISCAL_YEAR;
+import static org.folio.rest.utils.TestEntities.LEDGER;
+import static org.junit.Assert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+import io.restassured.response.Response;
+import io.vertx.core.json.JsonObject;
+import java.net.MalformedURLException;
+import org.folio.rest.persist.PostgresClient;
+import org.folio.rest.utils.TestEntities;
+import org.junit.Test;
+
+public class LedgerFYTest extends TestBase {
+
+  private static final String LEDGER_FY_TABLE = "ledgerFY";
+  private static final String LEDGER_FY_ENDPOINT = "/finance-storage/ledger-fiscal-years";
+
+  @Test
+  public void testGetQuery() throws MalformedURLException, InterruptedException {
+
+    String fiscalYearId = testPositiveCases(FISCAL_YEAR);
+    String ledgerId = testPositiveCases(LEDGER);
+
+    postLedgerFYData(fiscalYearId, ledgerId);
+
+    // search for GET
+    verifyCollectionQuantity(LEDGER_FY_ENDPOINT, 1);
+
+    // search with fields from "ledger"
+    getData(LEDGER_FY_ENDPOINT + "?query=ledger.name==NonExistent").then()
```

Comment:
You could simplify this check by using method `verifyCollectionQuantity` something like:
`verifyCollectionQuantity(LEDGER_FY_ENDPOINT, "?query=ledger.name==NonExistent", 0);`
It's up to you.